examples/tutorial_server_eventfilter.c

```diff
@@ -0,0 +1,282 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+#include <open62541/plugin/log_stdout.h>
+#include <open62541/server.h>
+#include <open62541/server_config_default.h>
+
+#include <signal.h>
+#include <stdlib.h>
+
+/**
+ * Generating events
+ * -----------------
+ * To make sense of the many things going on in a server, monitoring items can be useful.
+ * Though in many cases, data change does not convey enough information to be the optimal
+ * solution. Events can be generated at any time, hold a lot of information and can be
+ * filtered so the client only receives the specific attributes he is interested in.
+ *
+ * Emitting events by calling methods
+ * ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ * The following example is an extension of the server event tutorial. We will be
+ * creating two method nodes. One which generates multiple event from the server node and
+ * one which generates a random event
+ *
+ * The event we want to generate should not be simple, in order to allow the client to set
+ * a meaningful filter. `EventTypes` are saved internally as `ObjectTypes`,
+ * so add the a few types as you would new `ObjectTypes`. */
```

Comment:
This example can be used together with the event filter example. But the example itself contains no event filter specific definitions. Therefore please don't call it tuturial, mention the client example here and rename it e.g. server_random_events.c